public/tracy/TracyVulkan.hpp

```diff
@@ -255,6 +257,7 @@ class VkCtx
             return;
         }
 #endif
+        assert(m_head >= m_tail);
```

Comment:
There's also this access to `m_head` which should have memory order specfied. Moreover, value should be read before checking the assert, as the `m_head` value may change between the assert and the usage in line 270 below.